Doctrine/RouteConditionMetadataListener.php

```diff
@@ -49,6 +49,11 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs)
         }
 
         $meta = $eventArgs->getClassMetadata();
+
+        if (null === $meta->getReflectionClass()) {
```

Comment:
but did you see this happen in your project?

i am not against doing this, but i would like to understand when this
can happen to avoid hiding a different problem. maybe there is no other
solution and this is normal and can happen, but i am not sure.